test/topology_experimental_raft/test_fencing.py

```diff
@@ -80,10 +83,11 @@ async def test_fence_writes(request, manager: ManagerClient):
         Column("pk", IntType),
         Column('int_c', IntType)
     ])
-    table2 = await random_tables.add_table(name='t2', pks=1, columns=[
-        Column("pk", IntType),
-        Column('counter_c', CounterType)
-    ])
+    if not tablets_enabled:
```

Comment:
Nitpick: this doesn't have any reference to the issue number, so when we finally do support tablets, nobody will remember we need to fix this test back.

Reply:
Good catch, I added the issue reference to the both `if` statements I've introduced to this testcase.